query.go

```diff
@@ -155,6 +157,8 @@ func (q *Query) Filter(filterStr string, value interface{}) *Query {
 		f.Op = greaterThan
 	case "=":
 		f.Op = equal
+	case "array-contains":
```

Comment:
I don't think you'll ever get here. The strings.TrimRight above won't work for arraycontains. It probably needs to use TrimSuffix in the arraycontains case.

It'd be nice to add a query_test.go that creates a query with an arraycontains filter, and then make sure toProto returns the right operator.

Reply:
Potentially addressed with the HasSuffix block